torch/csrc/Storage.cpp

```diff
@@ -134,8 +134,8 @@ static PyObject* THPStorage_pynew(
   torch::ParsedArgs<3> parsed_args;
   auto r = parser.parse(args, kwargs, parsed_args);
 
-  int64_t allocator_arg_idx = 0;
-  int64_t device_arg_idx = 1;
+  int allocator_arg_idx = 0;
```

Comment:
Do we really want to lower it to an int?

Reply:
Or change interface to accept int64?

Reply:
The following API wants int  indices. We should revise them in another PR.